src/main/java/uk/ac/ebi/eva/pipeline/parameters/ParametersFromProperties.java

```diff
@@ -62,6 +62,12 @@
     @Value(PROPERTY + JobParametersNames.OUTPUT_DIR_STATISTICS + OR_NULL)
     private String outputDirStats;
 
+    @Value(PROPERTY + JobParametersNames.STATISTICS_SKIP + OR_NULL)
+    private String statisticsSkip;
+
+    @Value(PROPERTY + JobParametersNames.ANNOTATION_SKIP + OR_NULL)
```

Comment:
This is only to pass the value from the properties as jobParameters. It is only for Integration test and if no defined in the property file the behaviour defaults to false value.